Python_tasks/data_checks.py

```
@@ -4,36 +4,84 @@
 import constants
 from load_files import load_data, get_lumi_data
 from pokemonUtils import (get_form_name, get_form_pokemon_personal_id,
-                          get_pokemon_name, get_pokemon_name, get_diff_form_dictionary)
+                          get_pokemon_name, get_pokemon_name, get_diff_form_dictionary,
+                          get_pokemon_mons_no_from_name)
+
+def check_monsName(monsName):
+    '''
+    This checks that a pokemons name is formatted correctly specifically for the Honey trees
+    '''
+    monsNo = get_pokemon_mons_no_from_name(monsName)
+    if monsNo == -1:
+        print(monsName)
```

This should probably throw instead of just printing